models/src/commonMain/kotlin/com/revenuecat/purchases/kmp/models/StoreTransaction.kt

```diff
@@ -0,0 +1,26 @@
+package com.revenuecat.purchases.kmp.models
+
+/**
+ * Represents an in-app billing purchase.
+ */
+public data class StoreTransaction(
```

Comment:
We have a mix of data classes and regular classes. I think we should probably stick to regular classes since data classes adds some functions to the API, and making it a regular class in the future will be breaking